### DISCUSSION TITLE
Subscribe to new issues without watching entire repo

### DISCUSSION THREAD

**Discussion by dgaetjens** [community/community#23245] [New to GitHub]
<p>Is there a way to be notified of new issues that are opened on a github repository, without watching the entire repository?<br>
Specifically, I would like to be notified when new issues are raised, without being notified of other activities (such as when new pull requests are made, unless I participate or have been <span class="mention">@mentioned</span>). I am concerned that if I watch the entire repository, I may begin to overlook new issues due to being overwhelmed with other notifications. My current workaround is to add a <span class="mention">@mention</span> to myself in the repositories issue template. Would anyone be able to suggest a better way? I have been unable to find an appropriate setting in <a href="https://github.com/settings/notifications" rel="nofollow noopener">https://github.com/settings/notifications</a></p>

**Comment by dgaetjens** [comment, marked as answer]
<p>After further investigation, I have discovered that github webhooks can be set to only trigger on issues. Specifically when an issue is “opened, edited, deleted, transferred, pinned, unpinned, closed, reopened, assigned, unassigned, labeled, unlabeled, milestoned, demilestoned, locked, or unlocked”, with a different option for issue comments. I have linked the MS teams github enterprise integration to this webhook which fulfils my needs.</p>

**Comment by markcellus** [comment]
<p>Hello, I’m having trouble understanding this solution. Can you explain further?</p>

**Comment by dgaetjens** [comment]
<p>The solution was to install the MS teams GitHub Enterprise add-on (called a ‘connector’ on MS teams) to an MS teams channel. This MS teams add-on can post messages to an MS teams channel whenever something happens on a specified github repo, such as when a new pull request is opened, or someone posts a comment.</p>
<p>The add-on communicates to github using a webhook. The noise problem was solved by configuring the webhook to only be triggered when issue related events happen (under settings&gt;hooks&gt;Webhooks&gt;Edit&gt;Which events would you like to trigger this&gt;Let me select individual events).</p>
